include/openmrn_features.h

```diff
@@ -48,6 +48,10 @@
 #define OPENMRN_FEATURE_REENT 1
 #endif
 
+#if defined(__linux__) || defined(__MACH__) || defined(__WINNT__) || defined(ESP32) || defined(OPENMRN_FEATURE_DEVTAB)
+#define OPENMRN_HAVE_POSIX_FD 1
```

Comment:
for consistency please add a /// comment for what this define enables.

Reply:
good point, done.